docs/_config.yml

```diff
@@ -1,8 +1,8 @@
 name: LitElement
-description: A simple base class for creating fast, lightweight web components
+description: A simple base class for creating fast, <br/> lightweight web components
```

Comment:
No tags here - description is also used in `<meta name="description" content="{{ site.description }}">`